deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml

```diff
@@ -1,7 +1,7 @@
 channels:
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.11.0
   name: alpha
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.9.1
```

Comment:
shouldn't we have here 0.10.0 or 0.11.0 ?

Reply:
no, that is a dev-preview channel, it should point to DP version (0.9.1)

Reply:
oki :)